devbox.json

```diff
@@ -5,13 +5,11 @@
     "path:.#log-cache-cli-plugin":       "",
     "path:.#cloud-mta-build-tool":       "",
     "path:.#uaac":                       "",
-    "act":                               "0.2.64",
```

Comment:
It's not removed just downgraded. due to 
> The downgrade of act is due to https://github.com/nektos/act/issues/2426 which prevents the devbox action from correctly running in act

Further down in the file the act is added in version 0.2.63.

To be honest the fact that this file is not ordered, so that the diffs are meaningful has bothered me beofre.